src/engine/test/assets/decoders/aws_s3.yml

```diff
@@ -59,4 +83,4 @@ normalize:
 
       tls.cipher: $_json.aws.cipher_suite
 
-      json: +delete_field
+      _json: +delete_field
```

Comment:
This is not necessary since fields beginning with `_` will be automatically deleted at the end of the event processing.